docs/customization/suites.rst

```diff
@@ -0,0 +1,259 @@
+Customizing Suites
+==================
+
+What are suites:
+~~~~~~~~~~~~~~~~
+Suites are a pocket fixtures that are defined in one place.
+
+Why would you customize a suites?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you need to create a configuration file where you define all necessarily configuration to your store.
+
+How to use suites?
+~~~~~~~~~~~~~~~~~~
+List suites are available after ``$ bin/console sylius:fixtures:list`` command.
+In Sylius default suites are loaded in ``$ bin/console sylius:fixtures:load``.
+If you creating new suite you must use previous command with name of your suite: ``$ bin/console sylius:fixtures:load your_custom_suite``.
+
+How to create suites?
+~~~~~~~~~~~~~~~~~~~~~
+
+.. tip::
+
+    You can browse the full implementation of this example on `this GitHub Pull Request
+    <https://github.com/Sylius/Customizations/pull/24>`__.
+
+If you want create your fixtures with different locale than en_US you must change parameter in ``src/config/services.yaml``.
```

Comment:
```suggestion
.. warning::
	If you want create your fixtures with different locale than ``en_US`` you must change the ``locale`` parameter in ``src/config/services.yaml``.
	
	.. code-block:: yaml
	    parameters:
	        locale: pl_PL
```

Can you check, how does it look like?

Reply:
![Screenshot 2019-08-26 at 10 17 01](https://user-images.githubusercontent.com/39232096/63676194-b1286500-c7ea-11e9-91fb-9ab1e3920725.png)